journal-writings/modernized/06-chapter-4.adoc

```diff
@@ -660,4 +667,3 @@ From hence I was at three meetings in my way, and so went home,
 under a humbling sense of the gracious dealings of the Lord with me,
 in preserving me through many trials and afflictions in my journey.
 I was out about two months, and travelled about eleven hundred and fifty miles.
-
```

Comment:
we might need to get your Atom editor setting coordinated with mine. I have mine set to always have a newline at the end of every file.  It's generally recommended I think, because it cleans up diffs.

Reply:
Yeah, go to Preferences in Atom.  Then click on "Packages" and search for "whitespace", and then click it's settings gear.  Then check the `Ensure Single Trailing Newline` box (near the top).